CMakeLists.txt

```diff
@@ -20,6 +20,10 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT AND MSVC)
     set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "Default install dir on windows" FORCE)
 endif()
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Don't use this method. Instead, in the wpiutil build there is a flag to set the requirement on the wpiutil target, and any targets that inherit from it.

Reply:
https://github.com/wpilibsuite/allwpilib/blob/master/wpiutil/CMakeLists.txt#L82

Reply:
Done.